packages/app/lib/ui/widgets/image_carousel.dart

```diff
@@ -1,81 +1,145 @@
+    return MeliCard(
+      borderColor: MeliColors.black,
+      borderWidth: 3.0,
+      child: Container(
+        clipBehavior: Clip.hardEdge,
+        height: 200.0,
+        decoration: ShapeDecoration(
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(12),
+          ),
+        ),
+        child: Stack(
+          children: [
+            CarouselSlider(
+              carouselController: this._controller,
+              options: CarouselOptions(
+                  height: 200.0,
+                  onPageChanged: (value, reason) {
+                    setState(() {
+                      this._currentIndex = value;
+                    });
+                  },
+                  enableInfiniteScroll: false,
+                  viewportFraction: 1,
+                  padEnds: false,
+                  enlargeCenterPage: false),
+              items: this.widget.imagePaths.map((path) {
+                return CarouselItem(imagePath: path);
+              }).toList(),
+            ),
+            if (this.widget.onDelete != null) this._deleteButton(),
+            if (this._currentIndex > 0) this._previousButton(),
+            if (this._currentIndex < this.widget.imagePaths.length - 1)
+              this._nextButton(),
+          ],
+        ),
       ),
     );
   }
 }
 
 class CarouselItem extends StatelessWidget {
-  const CarouselItem(
-      {super.key, required this.image, required this.index, this.onDelete});
+  const CarouselItem({super.key, required this.imagePath});
 
-  final Image image;
-  final int index;
-  final Function? onDelete;
+  final String imagePath;
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-        width: double.infinity,
-        child: Stack(children: [
-          Container(
-            alignment: Alignment.center,
-            child: image,
-          ),
-          if (this.onDelete != null)
-            Container(
-              alignment: Alignment.bottomCenter,
-              child: ElevatedButton(
-                style: ButtonStyle(
-                    backgroundColor:
-                        MaterialStateProperty.all<Color>(Colors.red[800]!),
-                    shape: MaterialStateProperty.all<CircleBorder>(
-                        CircleBorder())),
-                onPressed: () {
-                  this.onDelete!(index);
-                },
-                child: Icon(
-                  Icons.delete_outlined,
-                  color: Colors.white,
-                ),
-              ),
-            )
-        ]));
+    BoxFit fit = BoxFit.cover;
+    double width = double.infinity;
+
+    if (this.imagePath.contains('http')) {
+      return Image.network(this.imagePath, fit: fit, width: width);
+    } else if (this.imagePath.startsWith('/')) {
+      return Image.file(File(this.imagePath), fit: fit, width: width);
+    } else {
+      return Image.asset(this.imagePath, fit: fit, width: width);
+    }
+  }
```

Comment:
I wonder if users will be bothered that their images get cropped when taken portrait? I don't see a great way around it unless we make changes to the design (square images?). I could imagine important bits of bees getting cropped off if users aren't careful.

What are your thoughts? Could open a separate issue if we wanna discus more as it doesn't need to hold up this pr.

Reply:
I think we should have a way to look at images fullscreen and not in this little widget. Can imagine that for proper identification that's too small.

Reply:
I've made a ticket!